### PR TITLE
fix(storage) update action_url, notes & notes_url size in centreon_storage (21.04)

### PR DESCRIPTION
**Pull request by a-launois**
## Description

update action_url, notes & notes_url size in centreon_storage

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [x] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x (master)

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/install/sql/centstorage/Update-CSTG-21.04.15.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ALTER TABLE services
+    MODIFY COLUMN `action_url` VARCHAR(2048) DEFAULT NULL,
+    MODIFY COLUMN `notes_url` VARCHAR(2048) DEFAULT NULL,
+    MODIFY COLUMN `notes` VARCHAR(512) DEFAULT NULL;
+
+ALTER TABLE hosts
```
Comment (bouda1) [inline www/install/sql/centstorage/Update-CSTG-21.04.15.sql:6]: You should also update resources table...

Comment (bouda1) [reply www/install/sql/centstorage/Update-CSTG-21.04.15.sql:6]:
```
desc resources;
+------------------------+---------------------+------+-----+---------+----------------+
| Field                  | Type                | Null | Key | Default | Extra          |
+------------------------+---------------------+------+-----+---------+----------------+
| resource_id            | bigint(20) unsigned | NO   | PRI | NULL    | auto_increment |
| internal_id            | bigint(20) unsigned | YES  |     | NULL    |                |
| id                     | bigint(20) unsigned | NO   | MUL | NULL    |                |
| parent_id              | bigint(20) unsigned | NO   |     | NULL    |                |
| type                   | tinyint(3) unsigned | NO   |     | NULL    |                |
| status                 | tinyint(3) unsigned | YES  |     | NULL    |                |
| status_ordered         | tinyint(3) unsigned | YES  |     | NULL    |                |
| in_downtime            | tinyint(1)          | NO   |     | 0       |                |
| acknowledged           | tinyint(1)          | NO   |     | 0       |                |
| status_confirmed       | tinyint(1)          | YES  |     | NULL    |                |
| check_attempts         | tinyint(3) unsigned | YES  |     | NULL    |                |
| max_check_attempts     | tinyint(3) unsigned | YES  |     | NULL    |                |
| poller_id              | bigint(20) unsigned | NO   |     | NULL    |                |
| severity_id            | bigint(20) unsigned | YES  | MUL | NULL    |                |
| name                   | varchar(255)        | YES  |     | NULL    |                |
| alias                  | varchar(255)        | YES  |     | NULL    |                |
| address                | varchar(255)        | YES  |     | NULL    |                |
| parent_name            | varchar(255)        | YES  |     | NULL    |                |
| notes_url              | varchar(255)        | YES  |     | NULL    |                |
| notes                  | varchar(255)        | YES  |     | NULL    |                |
| action_url             | varchar(255)        | YES  |     | NULL    |                |
| has_graph              | tinyint(1)          | NO   |     | 0       |                |
| notifications_enabled  | tinyint(1)          | NO   |     | 0       |                |
| passive_checks_enabled | tinyint(1)          | NO   |     | 0       |                |
| active_checks_enabled  | tinyint(1)          | NO   |     | 0       |                |
| last_check_type        | tinyint(3) unsigned | NO   |     | 0       |                |
| last_check             | bigint(20) unsigned | YES  |     | NULL    |                |
| last_status_change     | bigint(20) unsigned | YES  |     | NULL    |                |
| output                 | text                | YES  |     | NULL    |                |
| enabled                | tinyint(1)          | NO   |     | 1       |                |
| icon_id                | bigint(20) unsigned | YES  |     | NULL    |                |
+------------------------+---------------------+------+-----+---------+----------------+
31 rows in set (0.003 sec)
```

**Review by bouda1 — CHANGES_REQUESTED** [review]

**Review by bouda1 — APPROVED** [review]
OK for 21.04.
Be careful in 22.04 and next, to also update the resources table.

**Comment by a-launois** [comment]
Due to scope change this fix is not required on this version anymore

**Pull request closed without merging**